types/artifacts.d.ts

```diff
@@ -701,7 +712,8 @@ declare global {
 
       export interface Form {
         /** If attributes is missing that means this is a formless set of elements. */
-        attributes?: NodeDetails & {
+        attributes?: {
+          node: NodeDetails;
           id: string;
           name: string;
           autocomplete: string;
```

Comment:
FormInput and FormLabel both have nodeLabel and snippet, but left out incorporating NodeDetails, since it seems a bit too much, given that that information is already available on the Form artifact. Any thoughts?

Reply:
ah here's your comment about being selective.   the inputs/labels reference different dom nodes than the  Form's one.  So i still think it's worthwhile to keep all of `node` for inputs/labels.